<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dbEnumerations</class>
 <widget class="QMainWindow" name="dbEnumerations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>646</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>BiblioteQ: Database Enumerations Browser</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons.d/book.png</normaloff>icons.d/book.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QToolButton" name="saveButton">
        <property name="font">
         <font>
          <family>Times</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Save Changes</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons.d/32x32/save.png</normaloff>icons.d/32x32/save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="reloadButton">
        <property name="font">
         <font>
          <family>Times</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Refresh</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons.d/32x32/reload.png</normaloff>icons.d/32x32/reload.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="cancelButton">
        <property name="font">
         <font>
          <family>Times</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Close Browser</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons.d/32x32/cancel.png</normaloff>icons.d/32x32/cancel.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>saveButton</tabstop>
  <tabstop>reloadButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
